contracts/wombat-governance/interfaces/IBribe.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Per https://github.com/wombat-exchange/v1-core#licensing, interface should be GPL-3.0. THis is consistent with other interfaces. Also, please update upstream afterwards.